blocks/hero/hero.js

```diff
@@ -0,0 +1,73 @@
+import { toCamelCase } from '../../scripts/aem.js';
+import { div } from '../../scripts/dom-helpers.js';
+
+function readHeroBlockConfig(block) {
+  const config = {};
+
+  block.querySelectorAll(':scope > div').forEach((row) => {
+    if (row.children) {
+      const cols = [...row.children];
+      if (cols[1]) {
```

Comment:
you can shorthand this using optional chaining
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

```
if (row?.children?.[1] !== undefined)
```